packages/kit/src/runtime/client/client.js

```diff
@@ -471,19 +471,11 @@ export function create_client({ target, base, trailing_slash }) {
 	 *   url: URL;
 	 *   params: Record<string, string>;
 	 *   routeId: string | null;
-	 * 	 server_data_node: import('./types').DataNode | import('types').ServerErrorNode | null;
+	 * 	 server_data_node: import('./types').DataNode | null;
 	 * }} options
 	 * @returns {Promise<import('./types').BranchNode>}
 	 */
 	async function load_node({ loader, parent, url, params, routeId, server_data_node }) {
-		if (server_data_node?.type === 'error') {
-			if (server_data_node.httperror) {
-				throw error(server_data_node.httperror.status, server_data_node.httperror.message);
-			}
-
-			throw server_data_node.error;
-		}
```

Comment:
don't we need this? it doesn't seem to be accounted for in the other changes

Reply:
No we don't, this case is handled in line 750 and 888, and in line 1318 it can never be this kind of type. Furthermore, `server_data_node` will never be of this kind since `create_data_node`, the function which transforms it, would return `null` for error types.